src/test/java/nl/tudelft/ewi/devhub/webtests/views/PullRequestView.java

```diff
@@ -0,0 +1,17 @@
+package nl.tudelft.ewi.devhub.webtests.views;
+
+import org.openqa.selenium.WebDriver;
+
+public class PullRequestView extends AuthenticatedView {
+
+	public PullRequestView(WebDriver driver) {
+		super(driver);
+	}
+
+	@Override
+	protected void invariant() {
+		// TODO Auto-generated method stub
```

Comment:
That would definitely be a good idea. On the other hand, I would also like to create an abstraction for the (potential) `DiffInPullRequestView`, and `DiffInCommitView`, since diffs are quite complex. Maybe we can introduce a header field to provide interaction in both views?